### PR TITLE
icons: created custom icons using react-native-vector-icons

### PR DESCRIPTION
**Pull request by Nap1400**
Used FontAwesome5 library.

### CODE REVIEW

**Comment by Nap1400** [comment]
icons

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
   },
   "dependencies": {
     "react": "17.0.2",
-    "react-native": "0.67.3"
+    "react-native": "0.67.3",
+    "react-native-vector-icons": "^9.1.0"
```
Comment (Dulaya) [inline package.json:15]: Note: if you're using this component, you need to run `npx react-native link` to to let `react-native-vector-icons` library render icons properly.

File: src/components/icon/Icon.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {View,Text} from 'react-native';
+import Icons from 'react-native-vector-icons/FontAwesome5';
+import {styles} from './Icon.style';
+
+export const Icon = ({iconName = "angle-right",iconSize = styles.regular,iconColor = "#000000"}) => {
+    console.log(styles);
```
Comment (Dulaya) [inline src/components/icon/Icon.js:7]: Maybe get rid of console.log().

**Review by Dulaya — APPROVED** [review]
I tried these code, and it works fine on my end.

File: src/components/icon/Icon.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  return (
+  <View>
+    <Icons name={iconName} type="FontAwesome5" size={styles[iconSize]} color={iconColor} style={{margin: 5, }} />
+  </View>
```
Comment (lasharela) [inline src/components/icon/Icon.js:11]: never ever use inline styles.

File: src/components/icon/Icon.style.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import {StyleSheet} from 'react-native';
+
+export const styles = StyleSheet.create({
+    small: 40,
```
Comment (lasharela) [inline src/components/icon/Icon.style.js:4]: not relevant JSS. inside of styleSheet keep styles.

**Review by lasharela — CHANGES_REQUESTED** [review]

**Review by lasharela — APPROVED** [review]

**Review by raynesmike — APPROVED** [review]
Everything seem to work with the icons when on the top section

**Pull request merged**